src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/InnvilgeVedtak/InnvilgeOvergangsstønad.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback, useState } from 'react';
+import { IInnvilgeVedtakForOvergangsstønad } from '../../../../../App/typer/vedtak';
+import { RessursFeilet, RessursStatus, RessursSuksess } from '../../../../../App/typer/ressurs';
+import { useBehandling } from '../../../../../App/context/BehandlingContext';
+import { useApp } from '../../../../../App/context/AppContext';
+import { Behandling } from '../../../../../App/typer/fagsak';
+import { VEDTAK_OG_BEREGNING } from '../../Felles/konstanter';
+import { revurdererFraPeriodeUtenStønad } from './revurderFraUtils';
+import { RevurderesFraOgMed } from '../../Felles/RevurderesFraOgMed';
+import { IVilkår } from '../../../Inngangsvilkår/vilkår';
+import { InnvilgeVedtak } from './InnvilgeVedtak';
+import { oppdaterVedtakMedEndretKey } from './utils';
+
+export const InnvilgeOvergangsstønad: React.FC<{
+    behandling: Behandling;
+    lagretVedtak?: IInnvilgeVedtakForOvergangsstønad;
+    vilkår: IVilkår;
+}> = ({ behandling, lagretVedtak, vilkår }) => {
+    const { behandlingErRedigerbar } = useBehandling();
+    const { axiosRequest, settIkkePersistertKomponent } = useApp();
+    const [vedtak, settVedtak] = useState<IInnvilgeVedtakForOvergangsstønad | undefined>(
```

Comment:
Hadde en tanke her først om å bruke ressurs, men for å unngå dobbel feilmelding fra RevurderesFraOgMed og en eventuell Dataviewer så ble det sånn her. Men kan endre, og er åpen på forslag på andre forbedrelser :)